_events.yml

```diff
@@ -1,10 +1,18 @@
-- date: 2019-04-01
-  title: "Coinfest 2019"
-  venue: "MeeM"
-  address: "Coehoornstraat 8"
-  city: "Arnhem Bitcoin City"
-  country: "The Netherlands"
-  link: "https://www.arnhembitcoinstad.nl"
+- date: 2019-05-11
+  title: "Magical Crypto Conference"
+  venue: "Metropolitan Pavilion"
+  address: "125 West 18th Street"
+  city: "New York"
+  country: "United States"
+  link: "https://www.magicalcryptoconference.com/"
+  
+- date: 2019-06-03
+  title: "Value of Bitcoin Conference"
+  venue: "Bavarian State Bank - BayernLB"
+  address: "Brienner Straße 18"
+  city: "80333 Munich"
```

Comment:
Could you please move `80333` to the end of the `address` line? The `city` should just be `Munich`.

Reply:
ah yes. Done!

Reply:
Thanks!